src/firebase.ts

```diff
@@ -1,37 +1,38 @@
-import { initializeApp } from 'firebase/app'
+import { initializeApp } from 'firebase/app';
 import {
   deleteDoc,
   doc,
   DocumentData,
   getDoc,
   getFirestore,
   setDoc
-} from 'firebase/firestore'
+} from 'firebase/firestore';
 import {
   ImageMessage,
   TextMessage,
   User
-} from './types'
+} from './types';
 
 const firebaseConfig = {
-  apiKey: 'AIzaSyCaYDFqWQvrA9nYlg446x0XKHTejrWJea0',
-  authDomain: 'projekty-do-nauki.firebaseapp.com',
-  projectId: 'projekty-do-nauki',
-  storageBucket: 'projekty-do-nauki.appspot.com',
-  messagingSenderId: '532844117414',
-  appId: '1:532844117414:web:5d766275ce722a24d66367'
-}
+  apiKey: "AIzaSyCaYDFqWQvrA9nYlg446x0XKHTejrWJea0",
+  authDomain: "projekty-do-nauki.firebaseapp.com",
+  databaseURL: "https://projekty-do-nauki-default-rtdb.europe-west1.firebasedatabase.app",
+  projectId: "projekty-do-nauki",
+  storageBucket: "projekty-do-nauki.appspot.com",
+  messagingSenderId: "532844117414",
+  appId: "1:532844117414:web:5d766275ce722a24d66367"
+};
```

Comment:
at this point Im not checking code further, please check PR before sending it, there should be no semicolons and double brackets